msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -477,11 +477,13 @@ This item is an array (maximum of 16 elements) with all elements of the type `ob
 |`websiteUrl`|string|||The https:// URL to point to if a user opts to view in a browser.|
 |`searchUrl`|string|||The https:// URL to point to for a user's search queries.|
 |`scopes`|array of enums|3|✔️|Currently, static tabs support only the `personal` scope, which means it can be provisioned only as part of the personal experience.|
-|`context` | array of enums| 8|| The set of `contextItem` scopes where a [tab is supported](../../tabs/how-to/access-teams-context.md). Default: **[personalTab, channelTab, privateChatTab, meetingChatTab, meetingDetailsTab, meetingStage, meetingSidepanel, teamLevelApp]**.|
+|`context` | array of enums| 8|| The set of `contextItem` scopes where a [tab is supported](../../tabs/how-to/access-teams-context.md). Default: [`personalTab`, `channelTab`, `privateChatTab`, `meetingChatTab`, `meetingDetailsTab`, `meetingStage`, `meetingSidepanel`, `teamLevelApp`].|
```

Comment:
teamLevelApp is not part of the default list. You can provide 2 lists here: accepted values and default values. They are not the same thing:
Here's the list of accepted values: 
"personalTab", "channelTab", "privateChatTab","meetingChatTab", "meetingDetailsTab", "meetingSidePanel", "meetingStage","teamLevelApp"
and here's the list of default values when the value is not supplied:
personalTab, channelTab, privateChatTab, meetingChatTab, meetingDetailsTab.

Reply:
Updated as suggested